From e1d8a5f671bd8f6a20aac19e8c2cb43df7d14808 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Fri, 11 Dec 2015 09:19:36 -0500
Subject: [PATCH 0606/1110] drm/amd/dal: Log connector signal at creation

This will help us better understand what configuration
QA used for testing when they give us a dmesg log.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Acked-by: Harry Wentland<harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/core/dc_link.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/core/dc_link.c b/drivers/gpu/drm/amd/dal/dc/core/dc_link.c
index 2f2800f..5890555 100644
--- a/drivers/gpu/drm/amd/dal/dc/core/dc_link.c
+++ b/drivers/gpu/drm/amd/dal/dc/core/dc_link.c
@@ -747,8 +747,10 @@ static bool construct(
 	}
 
 	/* TODO: #DAL3 Implement id to str function.*/
-	LINK_INFO("Connector[%d] description:\n",
-			init_params->connector_index);
+	LINK_INFO("Connector[%d] description:"
+			"signal %d\n",
+			init_params->connector_index,
+			link->public.connector_signal);
 
 	link->connector = dal_connector_create(dc_ctx, as, link->link_id);
 	if (NULL == link->connector) {
-- 
2.7.4

